knowledge-base/number-in-the-cell-is-formatted-as-text-or-preceded-by-an-apostrophe.md

```diff
@@ -0,0 +1,56 @@
+---
+title: How to Avoid Warning in Excel The number in the cell is formatted as text or preceded by an apostrophe
+description: The number in the cell is formatted as text or preceded by an apostrophe warning appears in Excel might appear if you exported integer values which are indicated as String
```

Comment:
Consider rewriting, for example:
The number in the cell is formatted as text or preceded by an apostrophe. This warning might appear if you exported integer values which are indicated as String in Excel